sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/Directory.Build.props

```diff
@@ -3,4 +3,5 @@
     Add any shared properties you want for the projects under this package directory that need to be set before the auto imported Directory.Build.props
   -->
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+  <Import Condition="'$(IsMgmtClientLibrary)' == 'true'" Project="$(RepoRoot)/sdk/core/Azure.Core/src/Azure.Core.props" />
```

Comment:
Why add this to all the individual libraries as opposed to shared targets file? 

cc @allenjzhang